website/src/parsers/js/typescript-eslint-parser.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import pkg from 'typescript-eslint-parser/package.json';
+import SettingsRenderer from '../utils/SettingsRenderer';
+
+const ID = 'typescript-eslint-parser';
+
+const defaultOptions = {
+  range: true,
+  loc: false,
+  tokens: false,
+  comment: false,
+  tolerant: true,
+  useJSXTextNode: false,
+
+  ecmaFeatures: {
+    jsx: true,
+  },
+};
+
+const parserSettingsConfiguration = {
+  fields: [
+    'range',
+    'loc',
+    'tokens',
+    'comment',
+    'tolerant',
+    'useJSXTextNode',
+    {
+      key: 'ecmaFeatures',
+      title: 'ecmaFeatures',
+      fields: Object.keys(defaultOptions.ecmaFeatures),
+      settings:
+        settings => settings.ecmaFeatures || {...defaultOptions.ecmaFeatures},
+    },
+  ],
+  required: new Set(['range']),
+};
+
+export default {
+  ...defaultParserInterface,
+
+  id: ID,
+  displayName: ID,
+  version: pkg.version,
+  homepage: pkg.homepage,
+  locationProps: new Set(['loc', 'start', 'end', 'range']),
+
+  loadParser(callback) {
+    require(['typescript-eslint-parser'], callback);
+  },
+
+  parse(parser, code, options) {
+    return parser.parse(code, {...defaultOptions, ...options} );
+  },
+
+  nodeToRange(node) {
+    if (node.range) {
+      return [node.range[0], node.range[1]];
```

Comment:
Heh, that's what I get for doing super-focused edits :P Fixed in https://github.com/fkling/astexplorer/pull/242/commits/49f9ff92dd1e9021cf117608f2bf74f916e21cae